var/spack/repos/builtin/packages/grib-util/package.py

```diff
@@ -30,11 +31,13 @@ class GribUtil(CMakePackage):
     depends_on("w3emc +extradeps", when="@1.2.4:")
     depends_on("w3emc precision=4,d", when="^w3emc@2.10:")
     depends_on("w3nco", when="@:1.2.3")
-    depends_on("g2")
+    depends_on("g2@3.4.9:")
```

Comment:
when="@1.4.0:" ?

Reply:
and similar for earlier versions?

Reply:
D'oh, yep. Just fixed for 1.4.0, if it's alright I'll do the other versions in the next batch of NCEPLIBS recipe updates.